docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,125 @@
+---
+product: base
+title: React Modal component
+components: ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
+---
+
+# Modal
+
+<p class="description">The `Modal` component lets you create dialogs, popovers, lightboxes, and other elements that force the user to take action before continuing.</p>
+
+The `Modal` component renders its `children` node in front of a backdrop component. This lets you create an element that your users must interact with before continuing in the parent application.
+Features:
+
+- 💄 Manages modal stacking when more than one is needed
```

Comment:
I wonder why the lipstick emoji?

Reply:
I wondered the same thing @michaldudak 😅 

Maybe 🥞 makes more sense?

Reply:
Was like that on the MD page :D No strong opinion, 📚 may be a different good candidate.